### ISSUE TITLE
3 digit color codes not recognized correctly

### ISSUE THREAD

**Issue by HundredVisionsGuy** [HundredVisionsGuy/webcode-tk#27]
Color contrast fails when a color code just has three digits.

If the color is #fff (for example), the code should convert it to #ffffff.

#369 should be #336699

**Comment by HundredVisionsGuy** [comment]
fixed with commit `bdcc307`
